subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -962,7 +978,7 @@ void *radio_ccm_rx_pkt_set(struct ccm *ccm, u8_t phy, void *pkt)
 	mode = (CCM_MODE_MODE_Decryption << CCM_MODE_MODE_Pos) &
 	       CCM_MODE_MODE_Msk;
 
-#if defined(CONFIG_SOC_COMPATIBLE_NRF52X)
+#if defined(CONFIG_SOC_COMPATIBLE_NRF52X) || defined(CONFIG_SOC_SERIES_NRF53X)
```

Comment:
Just make it `!defined(NRF51x)`?

Reply:
Same argument as above @joerchan, making it !NRF51 makes this code default for any other SoC and also, not educative when reading the code. When reading the code, IMHO, it is better to be able to see which platforms can enable the compilation of a code block (not the reverse).

No strong opinion though. I can change that if you insist.

Reply:
Will be automatically resolved after merging #20304